lib/src/Uncertainty/Distribution/BetaFactory.cxx

```diff
@@ -75,7 +75,9 @@ Beta BetaFactory::buildAsBeta(const Sample & sample) const
 
   const Scalar mu = sample.computeMean()[0];
   const Scalar sigma = sample.computeStandardDeviation()[0];
-  return buildAsBeta(BetaMuSigma(mu, sigma, a, b).evaluate());
+  Beta result(buildAsBeta(BetaMuSigma(mu, sigma, a, b).evaluate()));
+  adaptToKnownParameter(sample, &result);
```

Comment:
it looks setDescription is missing here unlike the others

Reply:
done